packages/backend/src/payment-method/ilp/service.ts

```diff
@@ -245,20 +246,6 @@ async function pay(
   }
 }
 
-function fromJSONtoRatio(ratio: unknown): Pay.Ratio {
```

Comment:
not needed anymore, we don't use `additionalFields` quote field to get these ratio amounts